EIPS/eip-2024.md

```diff
@@ -0,0 +1,90 @@
+---
+eip: 2024
+title: Blake2b Ethereum Implementation
+author: James Hancock <@madeoftin> , Tjaden Hess <@tjade273> 
```

Comment:
The original PR had "Jay Graber" also listed as an author.

Reply:
Added, I didn't see a contact for him, so I left that part out.